docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -48,6 +48,35 @@ It is no longer allowed to have the same feature in both the `--features` and `-
 The usage of unversioned feature names, e.g. `docker`, in the `--features` list will allow for the most supported / latest feature version to be enabled for you. 
 If you need more predictable behavior across releases, reference the particular version you want instead, e.g. `docker:v1`.
 
+= User Profile Changes
+
+== Default validations
+
+Default user profile configuration comes with set of default validations for the basic predefined fields. Those validations were not present in previous versions when `declarative-user-profile` feature was disabled by default.
+If you have issues due the backwards compatibility, it is recommended that your users are updated according to latest validations. They are typically enforced to do so whenever they update their account
```

Comment:
Isn't better to just remove any validation as they are business-specific?

IIRC, without user profile you would end up with the same validations as you were using the default user profile configuration, no?